src/top/nn/shortcut.cc

```diff
@@ -0,0 +1,38 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file shortcut.cc
+ * \brief Property def of pooling operators.
+ */
+#include <nnvm/op.h>
+#include <nnvm/node.h>
```

Comment:
put most operators under top/vision, we can address comments in the https://github.com/dmlc/tvm/pull/911 first and update accordingly